src/main/java/ai/rev/speechtotext/models/asynchronous/SummarizationFormattingOptions.java

```diff
@@ -0,0 +1,35 @@
+package ai.rev.speechtotext.models.asynchronous;
+
+import com.google.gson.annotations.SerializedName;
+
+/** Summarization formatting options. * */
```

Comment:
is there a purpose for `* */`? I see it everywhere in this PR

Reply:
This is how autoformatted puts it

Reply:
I did not find anything relevant in Google. Also, I don't see it like this in other files. Even in some of your files there is a single star only

Reply:
Okay. Thanks auto formatter! :-)
I'll spend some more time on this.